DarkFlippers/unleashed-firmware
# New RFID(125) and iButton Fuzzers

# What's new

- Two fuzzer
- - iButton
- - low frequency RFID (125khz)
- visual style inspired by [iButton fuzzer](https://github.com/DarkFlippers/unleashed-firmware/tree/58338ff51f6f9857f39ef07d5eb4495cdc02290d/applications/external/ibtn_fuzzer)
- both fuzzers are written from scratch
- At the moment, the path to the dictionaries corresponds to the previously used ones
- Time delay now displays the actual delay in tenths of a second (100ms)
- Now the time delay is divided into idle time _(time delay)_ and emulation time
- - Idle time can be zero

# Verification 

- Check the loading of different keys and user dictionaries
- Check emulation on different readers with different idle/emulation time
- [x] Change the category and name before merging

# Checklist (For Reviewer)
